lisa/testsuite.py

```diff
@@ -181,6 +186,7 @@ def set_status(
             if new_status == TestStatus.RUNNING:
                 self._timer = create_timer()
             self._send_result_message(self.stacktrace)
+            self._send_provision_boot_message()
```

Comment:
The boot message is about node, not test cases. So it should be sent from environment setup, not test case running.